### PR TITLE
Adding handlebars 4.0.3 orbit

### PR DESCRIPTION
**Pull request by rasika90**

### CODE REVIEW

File: handlebars/4.0.3.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ ~ Copyright (c) 2015, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (thusithathilina) [inline handlebars/4.0.3.wso2v1/pom.xml]: year should be changed to 2016

File: handlebars/4.0.3.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            <groupId>org.mozilla</groupId>
+            <artifactId>rhino</artifactId>
+            <version>${org.mozilla.rhino.version}</version>
+        </dependency>
+
+        <!-- Logging System -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${org.slf4j.slf4j-api.verison}</version>
```
Comment (thusithathilina) [inline handlebars/4.0.3.wso2v1/pom.xml]: all dependencies should be optional

File: handlebars/4.0.3.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.github.jknack</groupId>
```
Comment (thusithathilina) [inline handlebars/4.0.3.wso2v1/pom.xml]: groupid should be like org.wso2.orbit.com.github.jknack

File: handlebars/4.0.3.wso2v1/pom.xml (hunk 1 above)
Comment (rasika) [inline handlebars/4.0.3.wso2v1/pom.xml]: fixed in c85fcdd14da524eeaeeec3717a511ddc2209d534

File: handlebars/4.0.3.wso2v1/pom.xml (hunk 3 above)
Comment (rasika) [inline handlebars/4.0.3.wso2v1/pom.xml]: fixed in c85fcdd14da524eeaeeec3717a511ddc2209d534

File: handlebars/4.0.3.wso2v1/pom.xml (hunk 2 above)
Comment (rasika) [inline handlebars/4.0.3.wso2v1/pom.xml]: fixed in c85fcdd14da524eeaeeec3717a511ddc2209d534

**Pull request merged**